Pyrado/pyrado/policies/recurrent/two_headed_rnn.py

```diff
@@ -46,7 +47,7 @@ class TwoHeadedRNNPolicyBase(TwoHeadedPolicy, RecurrentPolicy):
     """
 
     # Type of recurrent network. Is None in base class to force inheritance.
-    recurrent_network_type = None
+    recurrent_network_type: Optional[RNNBase] = None
```

Comment:
I disagree here, since for the subclasses, the type must be a subclass of RNNBase, and must not be None